source/invoker.js

```diff
@@ -27,6 +27,16 @@ import toString from './toString';
  *      sliceFrom(6, 'abcdefghijklm'); //=> 'ghijklm'
  *      const sliceFrom6 = R.invoker(2, 'slice')(6);
  *      sliceFrom6(8, 'abcdefghijklm'); //=> 'gh'
+ *
+ *      const dog = {
+ *        speak: async () => 'Woof!'
+ *      };
+ *      const person = {
+ *        speak: async () => 'Hi!'
+ *      };
+ *      const speak = R.invoker(0, 'speak');
+ *      speak(dog).then(R.identity) //=> 'Woof!'
+ *      speak(person).then(R.identity) //=> 'Hi!'
```

Comment:
`.then(R.identity)` does literally nothing. I'd personally use `.then(console.log)` in the examples.

Reply:
Having console logs in examples is not very common in Ramda and that’s why I went with identity. If you look above to the slice example, the value returned is what’s in the comment- same applies here.

Reply:
Having console logs in examples is not very common in Ramda and that’s why I went with identity. If you look above to the slice example, the value returned is what’s in the comment- same applies here.

Reply:
I have no problem with `console.log` in an example.  It is rare in Ramda because our pure functions should return something that w)